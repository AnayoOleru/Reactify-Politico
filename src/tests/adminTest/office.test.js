import React from 'react';
import expect from 'expect';
import { shallow } from 'enzyme';
import { Office } from '../../components/pages/adminPages/office.jsx'

describe('Office component should', () => {
    const props = {
        CreateOffice: jest.fn(),
        getAllOffice: jest.fn(),
    };
    const wrapper = shallow(<Office {...props} />);
  it('renders without crashing', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
