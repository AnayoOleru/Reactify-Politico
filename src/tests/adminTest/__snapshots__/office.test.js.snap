// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Office component should renders without crashing 1`] = `ShallowWrapper {}`;
